.github/workflows/build.yml

```diff
@@ -62,6 +62,18 @@ jobs:
             type=semver,pattern={{major}}
             type=sha
 
+      - name: Prepare args
+        id: args
+        run: |
+          VERSION=edge
```

Comment:
The `edge` version is not used anymore in the product. It is fully-replaced by `master`, so please remove this line